### ISSUE TITLE
User data deletion

### ISSUE THREAD

**Issue by 54m4d** [spliit-app/spliit#186]
Hi! I really like Spliit! However, How can the user delete a group and/or its data after an event has finished?

This is important; otherwise, the group/data will still be there if the user visits the shared URL again.

There is #9. However, there is not much information on how groups/data are deleted.

To be honest, this is preventing me from continuing to use Spliit and I hope it can be sorted out soon.

Thanks!

**Comment by lorenzoalali** [comment]
I agree completely. I was very surprised not to find a way to delete a group once I was finished with it. I had to manually delete all the entries and anonymize all the information.

If the concern is that anyone could delete the group because there's no authentication, that's not a valid objection in my opinion, because anyone with malicious intent who gains access to the group can already delete everything.

What's even more troubling is that https://spliit.app/groups/xxx/activity keeps the names of deleted entries. This is by design, but until we have a proper login system (which requires more effort to implement than a simple delete button, and I am not expecting this to be on the roadmap), we urgently need a way to delete a group, as keeping entered data without a way to remove it is against good privacy practices, GDPR and other regulations.

In the meantime, thanks to the developers for making this tool!

**Comment by Monoxide8969** [comment]
Yeah, definitely a needed feature. Would be good for privacy and also reduce hosting costs I assume
